library/build.gradle

```diff
@@ -29,3 +43,24 @@ dependencies {
 }
 
 apply from: "${rootDir}/gradle/style.gradle"
+
+bintray {
+    user = System.getenv("BINTRAY_USER")
+    key = System.getenv("BINTRAY_KEY")
+    configurations = ["archives"]
+    pkg {
+        repo = "com.okta.android"
+        name = "okta-appauth"
+        desc = "Okta Authentication using AppAuth"
```

Comment:
@omgitstom was this what you wanted? I thought you had wanted `okta-sdk-appauth-android`

Reply:
Difference:
```
compile 'com.okta.android:appauth-android:0.1.0'
```
verses
```
compile 'com.okta.android:okta-sdk-appauth-android:0.1.0'
```

Reply:
I'm partial to `appauth-android`, since this has less redundant name spacing.

Reply:
Updated